nd4j/nd4j-backends/nd4j-api-parent/nd4j-api/src/main/java/org/nd4j/autodiff/validation/listeners/NonInplaceValidationListener.java

```diff
@@ -128,12 +128,12 @@ public void opExecution(SameDiff sd, At at, MultiDataSet batch, SameDiffOp op, O
                     "for op %s - input %s", op.getOp().getClass(), i);
 
             //Deallocate:
-            if(dealloc && after.closeable()) {
+         /*   if(dealloc && after.closeable()) {
                 after.close();
-            }
-            if(opInputs[i].closeable()){
+            }*/
+          /*  if(opInputs[i].closeable()){
                 opInputs[i].close();
-            }
+            }*/
```

Comment:
let's not have commented out code. If this is just temporary, put at least a TODO there, so it is obvious that this is still work in progress. Otherwise it is easy to miss this when you do the cleanup.